### ISSUE TITLE
Idea: graphql interface

### ISSUE THREAD

**Issue by aboodman** [rocicorp/replicache-client#4]
From https://github.com/aboodman/replicant/issues/146:

We frequently get asked about a graphql interface to noms/replicant. Until now I couldn't imagine what this would look like. @venkatd had a good idea -- it can just delegate to the transaction functions. We can iterate all the exported transaction functions. Without something like typescript, we don't know their signatures, but we know they can only take JSON types.

Basically you can think of the graphql API as an alternate autogenerated API for the transaction functions. This would be useful for customers who already have code that speaks graphql.

Going further, maybe a way to think of the transaction functions is as the implementation of the graphql interface. Maybe the graphql interface is the "real" interface.
